astropy/time/core.py

```diff
@@ -1928,19 +1928,19 @@ def now(cls):
         method is called.
 
         .. note::
-            "Now" is determined using the `~datetime.datetime.utcnow`
+            "Now" is determined using the `~datetime.datetime.now`
             function, so its accuracy and precision is determined by that
             function.  Generally that means it is set by the accuracy of
-            your system clock.
+            your system clock. Timezone is set to UTC.
 
         Returns
         -------
         nowtime : :class:`~astropy.time.Time`
             A new `Time` object (or a subclass of `Time` if this is called from
             such a subclass) at the current time.
         """
-        # call `utcnow` immediately to be sure it's ASAP
-        dtnow = datetime.utcnow()
+        # call `now` immediately to be sure it's ASAP
+        dtnow = datetime.now(tz=UTC)
```

Comment:
Not sure where you got the "UTC" but that was not the replacement that I found.
```suggestion
        dtnow = datetime.now(tz=timezone.utc)
```

Reply:
It is available [since 3.11](https://docs.python.org/3/library/datetime.html#datetime.UTC) as an alias for `timezone.utc`. If you folks prefer using the fully qualified name, I'll use it then.

Reply:
We currently support Python 3.9+ so we cannot use Python 3.11 only features.

Reply:
Ah, thanks for mentioning this. I have updated the PR to use only ' timezone.utc'  instead of 3.11's `UTC`.